app/Actions/Import/Exec.php

```diff
@@ -276,14 +295,14 @@ public function do(
 			// Album creation per directory
 			foreach ($dirs as $dir) {
 				$this->assertImportNotCancelled();
+				/** @var Album|null */
 				$album = $this->importMode->shallSkipDuplicates() ?
 					Album::query()
-						->select(['albums.*'])
-						->join('base_albums', 'base_albums.id', '=', 'albums.id')
-						->where('albums.parent_id', '=', $parentAlbum?->id)
-						->where('base_albums.title', '=', basename($dir))
-						->get()
-						->first() :
+					->select(['albums.*'])
+					->join('base_albums', 'base_albums.id', '=', 'albums.id')
+					->where('albums.parent_id', '=', $parentAlbum?->id)
+					->where('base_albums.title', '=', basename($dir))
+					->first() :
```

Comment:
@ildyria I have checked with cs-fixer. It is not cs-fixer which "destroys" the indention. `make formatting` is not responsible for that. Are you sure that your IDE (Visual Studio, I believe) is not somehow responsible for that? Maybe some option which "fixes" indention upon opening? I will fix the indention myself once again. But in the long run this gets really annoying.

Reply:
I will double check what is going on in my VScode.

Reply:
should be fixed.